apollo-normalized-cache-api/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/api/Record.kt

```diff
@@ -31,14 +31,32 @@ class Record(
   var date: Map<String, Long?>? = null
     private set
 
+  /**
+   * The arguments of the field this Record represents in its parent.
+   */
+  @ApolloExperimental
+  var arguments: Map<String, Any?> = emptyMap()
```

Comment:
As discussed, I think this should be a `Map<Map<String, Any?>>` to have the arguments per-field. Also there's always the case of list of scalars when you don't have a Record